src/zenml/cli/metadata_store.py

```diff
@@ -84,6 +88,40 @@ def list_metadata_stores() -> None:
     )
 
 
+@metadata_store.command(
+    "describe",
+    help="Show details about the current active metadata store.",
+)
+@click.argument(
+    "metadata_store_name",
+    type=click.STRING,
+    required=False,
+    default="",
+)
+def describe_metadata_store(metadata_store_name: str) -> None:
```

Comment:
I think it would be nicer to make this an `Optional[str]` and remove the default value in the click argument, and then set a default value if it's not passed similar to how you do it already: 
```python
metadata_store_name = metadata_store_name or repo.get_active_stack().metadata_store_name
```